neqo-bin/src/client/mod.rs

```diff
@@ -554,6 +557,18 @@ pub async fn client(mut args: Args) -> Res<()> {
             remote_addr,
         );
 
+        let migration = if args.shared.qns_test == Some("connectionmigration".to_owned()) {
+            remote_addrs.next().map_or_else(
+                || {
+                    qerror!("No migration address found for {host}");
+                    exit(127);
+                },
```

Comment:
Clippy suggested map_or_else, didn't it?  It's unfortunate, because this is probably better as:

```rust
if Some(addr) = remote_addrs.next() {
    Some((real_local.port(), addr))
} else {
    qerror!("Cannot migrate from {host} when there is no address that follows");
    exit(127);
}
```